### ISSUE TITLE
.contrib.charmsupport.nrpe.NRPE.write to preserve old entries in the monitors key

### ISSUE THREAD

**Issue by auria** [juju/charm-helpers#312]
When running the following, all entries on the monitors keymap will be overrid.
```
nrpe_setup = NRPE()
nrpe_setup.add_check(shortname='test', description='test desc', check_cmd='')
nrpe_setup.write()
```

The PR on https://github.com/juju/charm-helpers/pull/311 will preserve old entries provided nrpe_setup.remove_check(shortname='test') has not been called. Otherwise, it will remove the expected checks but it will preserve the rest.

**Comment by auria** [comment]
This patch has been merged into master, but it has not been released yet.

**Comment by auria** [comment]
Released on 0.19.14
